### DISCUSSION TITLE
「开源之史」系列之一/六：从反主流文化到未知的默认文化 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#42] [Announcements]
# 「开源之史」系列之一/六：从反主流文化到未知的默认文化 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/04-from-counterculture-to-unkonwculture/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

感觉刚刚开头就结束了?

其实吧, 并不是所有创始人都去了美国,
而是创造合格 FLOSS 工程后, 
都被美国资本请去了美国,

在其它国家暂时还没有对应完备的整体发展流水线;
以及对应灵活的移民政策...

另外, 说文化, 其实, 当前所有主流文化, 当初都是反主流文化,
简单的说, 一切顺应主流文化的文化, 根本就丧失了文化本质, 无法识别成独立文化,
也只有特立独行的文化, 才可能从主流文化中吸引先进人物, 
持续发展和主流文化交战, 直到替代或是和解...

**Reply by lijiangsheng1** [reply]
1/6 ,第六章的第一篇～

**Comment by zhuangbiaowei** [comment]
关于“为何绝大多数的开源项目创始人都去了美国？”，提供两条线索：

1. 《蓝图》这本书中提到，美国是世界上乌托邦实验最多的国家。
2. 霍炬的一篇文章《围绕RMS、开源、自由软件的两场“战争”》——“阿斯伯格综合征和黑客文化”这一节，提到当年的美国，对于RMS这样的人的宽容。

另外关于为何在国内，不会崇尚“Upstream first”？

我的思考是：中国从近代以来，积贫积弱。所谓自强运动，就是洋人有的我们也要有。中国人民要站起来，就是要国产替代，要100%自研。这样的思维，就不会赞赏“站在巨人肩膀上”的人，只会赞赏“自己变成巨人”的人。

**Reply by ZoomQuiet** [reply]
#是也乎,(￣▽￣)

主要还是傲慢吧, 中国文化太早熟了, 

> 天人合一

我们从来都是自己就是完整天下的, 自古能人太多,
如果不是完备开创一个独立领域的, 
很难值得推崇, 
比如 神农之后, 虽然医药发展了无数代,
感觉就没因为一生行医没出过事故就变成英雄的人物...

在中国, 其实也很 宽容 的 ,嘦不反对 ZF 无论怎么折腾, 好象没什么机制来立即压制你咯...
